diff --git a/node_modules/react-native-vibration/android/build.gradle b/node_modules/react-native-vibration/android/build.gradle
index 6fecf80..910ac8b 100644
--- a/node_modules/react-native-vibration/android/build.gradle
+++ b/node_modules/react-native-vibration/android/build.gradle
@@ -13,5 +13,5 @@ android {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:0.12.+'
+    implementation "com.facebook.react:react-native:0.12.+"
 }
