diff --git a/node_modules/react-native-vibration/android/build.gradle b/node_modules/react-native-vibration/android/build.gradle
index 6fecf80..7c70a4e 100644
--- a/node_modules/react-native-vibration/android/build.gradle
+++ b/node_modules/react-native-vibration/android/build.gradle
@@ -1,17 +1,17 @@
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 33
+    buildToolsVersion "33.0.0"
 
     defaultConfig {
         minSdkVersion 16
-        targetSdkVersion 23
+        targetSdkVersion 33
         versionCode 1
         versionName "1.0"
     }
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:0.12.+'
+    implementation "com.facebook.react:react-native:0.12.+"
 }
